docs/docs/webchat/webchat-button-disable.md

```diff
@@ -0,0 +1,89 @@
+---
+id: webchat-button-disable
+title: Enabling and Disabling Buttons
+---
+
+This functionality allows you to disable a button once the user has clicked on it. Thanks to this visual information, the user can better understand the logical flow and navigate more easily.
+
+To add this property by default, you must set `autodisable: true` in the webchat `index.js`.
+The button style properties must be set as well.
```

Comment:
The button style properties must be set as well, is wrong, we provide a default style of a disabled button, but they can be customized with disabledstyle